src/utils/zcl_abapgit_path.clas.abap

```diff
@@ -125,7 +125,7 @@ CLASS ZCL_ABAPGIT_PATH IMPLEMENTATION.
       ev_filename = iv_fullpath.
     ENDIF.
 
-    ev_filename = cl_http_utility=>unescape_url( escaped = ev_filename ).
+    ev_filename = cl_http_utility=>unescape_url( escaped = to_lower( ev_filename ) ).
```

Comment:
The class might be used elsewhere and should not be changed. 

Change the filename to lower in src/ui/pages/zcl_abapgit_gui_page_stage.clas.abap